.github/workflows/ci.yml

```diff
@@ -34,7 +34,12 @@ jobs:
           channel: 'stable'
       - run: dart --version
       - run: flutter --version
-      - run: flutter analyze
+      - name: analyze
+        run: |
+          cd example
+          flutter pub get
```

Comment:
In my pc, running flutter pub get runs it for both libadwaita and example so I think cd is not needed, global `flutter pub get` would work too.

Reply:
On my machine I have the same behavior, but the CI seems to behave differently. You can check one of the earlier failed attempt :(

Reply:
Ok 👍

Reply:
I agree it's sad :(